app/javascript/menu/group-switcher.jsx

```diff
@@ -0,0 +1,55 @@
+import Dropdown from 'carbon-components-react/es/components/Dropdown';
+import React from 'react';
+import SideNavItem from 'carbon-components-react/es/components/UIShell/SideNavItem';
+import { Collaborate20 } from '@carbon/icons-react';
+
+const { miqChangeGroup } = window;
+
+export const GroupSwitcher = ({ miqGroups, currentGroup, expanded }) => {
+  const options = miqGroups.map((g) => ({
```

Comment:
What about destructuring, then you don't need the g and this can be a one-liner.